lib/environment/foreground/ajax.js

```diff
@@ -108,12 +116,6 @@ function buildRequestParams({ method = 'GET', url, query = {}, headers = {}, dat
 	if (sameOrigin) {
 		// Add `app=res` to same-origin request URLs
 		urlObj.searchParams.set('app', 'res');
-
-		if (method !== 'GET' && method !== 'HEAD') {
-			// Send modhash for same-origin non-GET requests
-			const hash = loggedInUserHash();
-			if (hash) headers['X-Modhash'] = hash;
-		}
```

Comment:
This could have stayed in `buildRequestParams` if I made it async. too. Think I should move it back?